PatternPal/PatternPal.Core/RecognizerRunner.cs

```diff
@@ -103,9 +103,10 @@ public class RecognizerRunner
 
             foreach (IEntity entity in entities)
             {
-                bool result = check.Check(
+                //TODO make correct implementation using priorities
```

Comment:
Why did you do this? I know this implementation is incorrect, but it is handy to test things.